test/index.js

```diff
@@ -745,7 +744,7 @@ describe('Multiple connections', function () {
 
     it('should load all the servers routes', function (done) {
 
-        server.inject('/docs', function (res) {
+        server.connections[0].inject('/docs', function (res) {
```

Comment:
This was a breaking change in hapi. Next master build will fail as well without this change. I did this here because PRs that are red are harder to get merged :)